crates/orderbook/src/arguments.rs

```diff
@@ -1,18 +1,19 @@
-use anyhow::{anyhow, Context, Result};
-use model::app_id::AppId;
 use primitive_types::{H160, U256};
 use reqwest::Url;
 use shared::{
-    arguments::display_option, bad_token::token_owner_finder, fee_subsidy::cow_token::SubsidyTiers,
-    http_client, price_estimation::PriceEstimatorType, rate_limiter::RateLimitingStrategy,
+    arguments::display_option, bad_token::token_owner_finder, http_client,
+    price_estimation::PriceEstimatorType, rate_limiter::RateLimitingStrategy,
 };
-use std::{collections::HashMap, net::SocketAddr, num::NonZeroUsize, time::Duration};
+use std::{net::SocketAddr, num::NonZeroUsize, time::Duration};
 
 #[derive(clap::Parser)]
 pub struct Arguments {
     #[clap(flatten)]
     pub shared: shared::arguments::Arguments,
 
+    #[clap(flatten)]
+    pub shared_order_creation: shared::arguments::OrderCreationArguments,
```

Comment:
nit: From the perspective of the `orderbook` it doesn't matter that the arguments for the order creation are shared so I think it would be nice to drop that prefix. I think we only had it for the generic shared arguments because we didn't have a better name.